pyproject.toml

```diff
@@ -43,5 +43,5 @@ packages = ["_nx_parallel", "nx_parallel",]
 line-length = 88
 target-version = 'py310'
 
-[tool.ruff.per-file-ignores]
-"__init__.py" = ['I', 'F403']
+[tool.ruff.lint]
+per-file-ignores = { "__init__.py" = ['I', 'F403'] }
```

Comment:
Updated this to get rid of the following warning while running the update script:
```.sh
(base) aditi@Aditis-MacBook-Air nx-parallel % sh /Users/aditi/Desktop/nx-parallel/_nx_parallel/script.sh
warning: The top-level linter settings are deprecated in favour of their counterparts in the `lint` section. Please update the following options in `pyproject.toml`:
  - 'per-file-ignores' -> 'lint.per-file-ignores'
1 file reformatted
```

Reply:
It looks like the linters are now passing. :tada: